src/metadata/modules.cpp

```diff
@@ -484,11 +484,29 @@ static HRESULT LoadSymbols(IMetaDataImport *pMD, ICorDebugModule *pModule, VOID
     IfFailRet(pModule->GetBaseAddress(&peAddress));
     IfFailRet(pModule->GetSize(&peSize));
 
+    std::vector<unsigned char> peBuf;
+    ULONG64 peBufAddress = 0;
+    if (isInMemory)
+    {
+        ICorDebugProcess* process = 0;
+        IfFailRet(pModule->GetProcess(&process));
```

Comment:
https://github.com/dotnet/runtime/blob/d06742c2dbf9ded9c1a896c2a6d0ba1e333a906e/src/coreclr/debug/di/module.cpp#L1227
`GetProcess()` will `AddRef` for `ICorDebugProcess` internal object, should be "released" after end of usage.
Since you have several exit points (`IfFailRet`, `return E_FAIL;` and end of scope), you probably should use `ToRelease<ICorDebugProcess> process;` instead, as we usually do in code for COM objects... or care about COM object release at each exit points.

Reply:
You're right I completely missed that, I added the tag now.